lib/validators/phony_validator.rb

```diff
@@ -9,6 +9,7 @@ def validate_each(record, attribute, value)
 
     @record = record
     value = PhonyRails.normalize_number(value.dup, default_country_code: normalized_country_code) if normalized_country_code
+    value, = PhonyRails.extract_extension(value)
```

Comment:
it's equivalent to `value, _ = PhonyRails.extract_extension(value)`. may look creepy though